src/components/sidebarContent/UserProfile.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <SettingsCard title="your profile">
+  <ContentTitle title="your profile">
```

Comment:
But it's not just a title. It contents title but also contents everything what we put in a slot.